FAQ.md

```diff
@@ -207,3 +208,33 @@ become visible if you hover over the button with your cursor.
 You can also use the refresh icon button on the top right of the screen to achieve
 the same result:
 ![pull-request-ci-workflow-details-page-2.png](./docs/assets/ci-single-job-re-run-tutorial/pull-request-ci-workflow-details-page-2.png)
+
+
+## 14. How to fix the script configure, stuck at `RequestError: connect ETIMEOUT 2606:4700:6810:223:443` (or similar ports)?
+
+If you are having errors while running the configure script, similar to this:
+
+```
+YN0001: | RequestError: connect ETIMEOUT 2606:4700:6810:223:443
+at ClientRequest.<anonymous> (/home/ubuntu/cacti/.yarn/releases/yarn-4.1.0.cjs:195:14340)
+at Object.onceWrapper (node:events:632:26)
+at ClientRequest.emit (node:events:529:35)
+at o.emit (/home/ubuntu/cacti/.yarn/releases/yarn-4.1.0.cjs:190:90286)
+at TLSSocket.socketErrorListener (node:events:501:9)
+at TLSSocket.emit (node:events:517:28)
+at emitErrorNT (node:internal/streams/destroy:151:8)
+at emitErrorCloseNT (node:internal/streams/destroy:116:3)
+.
+.
+```
+
+Sometimes the IPv6 defaults can lead to npm registry being undiscoverable. To fix this issue, one can safely disable IPv6 defaults. One such way (via a different configuration file) to achieve this is:
```

Comment:
@jagpreetsinghsasan Please include information on which distro this fix was tested (I assume Ubuntu). Also, it would be nice to include a warning that it changes how ipv6 is handled in entire system, so it's quiet major and may affect other applications (someone could expect that it's a magic fix for npm only).

Reply:
Yes, I have added the environment where it is tested and the necessary warning for the same.